packages/maskbook/src/plugins/Collectible/SNSAdaptor/Collectible.tsx

```diff
@@ -301,7 +302,7 @@ export function Collectible(props: CollectibleProps) {
                     </div>
                 </CardActions>
             </CollectibleCard>
-            {asset.value?.end_time && isValidDate(asset.value.end_time) && (
+            {asset.value?.end_time && isValidDate(asset.value.end_time) && isAfter(asset.value.end_time, new Date()) && (
```

Comment:
`asset.value?.end_time && isValidDate(asset.value.end_time) && isAfter(asset.value.end_time, new Date())` abstract this into a semantic variable, like `const isOnSafe = ...`